app/assets/stylesheets/forever_style_guide/modules/_nav.scss

```diff
@@ -358,26 +355,22 @@ a.navigation-user_info-logged_out-log_in_link {
   }
 }
 
-// Overriding button styles to make it look good with smaller font size
-.navigation-user_info-logged_out-sign_up_btn {
-  margin-left: $padding-small-horizontal;
-  height: auto !important;
+.navigation-user_info-logged_in {
+  height: 100%;
+  margin-top: 7px;
+  margin-right: 15px;
 
-  @media screen and (min-width: $screen-md) {
-    font-size: $font-size-100 !important;
-    letter-spacing: $letter-spacing-large;
+  &.open {
+    .user_info-logged_in-account_dropdown_trigger-icon {
+      transform: rotate(180deg);
+    }
   }
 
-  @media screen and (min-width: $screen-lg) {
-    margin-left: $padding-large-horizontal;
-    font-size: $font-size-200 !important;
+  @media (max-width: $screen-mobile-menu) {
+    display: none; // hides dropdown on mobile
   }
 }
 
-.navigation-user_info-logged_in {
-  height: 100%;
-  margin-left: $padding-default-horizontal;
-}
 
 .user_info-logged_in-account_dropdown_trigger {
```

Comment:
all of these user_info classes should be namespaced more consistently and probably moved to a standalone module.